vars/infra.groovy

```diff
@@ -159,14 +165,8 @@ Object withArtifactCachingProxy(boolean useArtifactCachingProxy = true, Closure
  * @see withArtifactCachingProxy
  */
 Object runMaven(List<String> options, String jdk = '8', List<String> extraEnv = null, Boolean addToolEnv = true, Boolean useArtifactCachingProxy = true) {
-  List<String> mvnOptions = ['--batch-mode', '--show-version', '--errors', '--no-transfer-progress']
+  List<String> mvnOptions = ['--batch-mode', '--show-version', '--errors']
```

Comment:
```suggestion
  List<String> mvnOptions = ['--batch-mode', '--show-version', '--errors', '--no-transfer-progress']
```

Reply:
Confirmed the correct download from the proxy in https://ci.jenkins.io/job/Plugins/job/jenkins-infra-test-plugin/job/lemeurherve-patch-2/2/console